FSiAdminSecurityBundle.php

```diff
@@ -21,6 +22,22 @@ public function build(ContainerBuilder $container)
         parent::build($container);
 
         $container->addCompilerPass(new FirewallMapCompilerPass());
+
+        $doctrineConfigDir = realpath(__DIR__ . '/Resources/config/doctrine');
+
+        $mappings = array(
+            $doctrineConfigDir . '/model' => 'FSi\Bundle\AdminSecurityBundle\Security\Model',
+        );
+
+        $container->addCompilerPass(DoctrineOrmMappingsPass::createXmlMappingDriver($mappings));
+
+        if (class_exists('FOS\UserBundle\Entity\User')) {
```

Comment:
what if fos-user is in vendors, but it is not enabled bundle? I don't really know if this is case we need to care here